packages/reference/src/common/useEditorPermissions.ts

```diff
@@ -50,25 +50,29 @@ export function useEditorPermissions(props: EditorPermissionsProps) {
     }
 
     async function checkLinkAccess() {
-      if (entityType === 'Asset') {
-        const canRead = await canOnEntity('read', 'Asset');
-        setCanLinkEntity(canRead);
-      }
-      if (entityType === 'Entry') {
-        setCanLinkEntity(readableContentTypes.length > 0);
+      if (props.allContentTypes?.length) {
+        if (entityType === 'Asset') {
+          const canRead = await canOnEntity('read', 'Asset');
+          setCanLinkEntity(canRead);
+        }
+        if (entityType === 'Entry') {
+          setCanLinkEntity(readableContentTypes.length > 0);
+        }
+      } else {
+        setCanCreateEntity(true)
```

Comment:
You may want to set `setCanLinkEntity(true)` here as well, don't you?

Reply:
@anho uh, good catch, It should actually only be `setCanLinkEntity`  ;)